metafacture-xml/src/main/java/org/metafacture/xml/SimpleXmlEncoder.java

```diff
@@ -218,6 +216,29 @@ public void setNamespaces(final Map<String, String> namespaces) {
         this.namespaces = namespaces;
     }
 
+    /**
+     * Sets the namespace(s).
+     *
+     * @param namespacesString the namespaces as a String. Key and value are separated by a '='. Multiple entries are
+     *                       separated by a line feed '\n'.
+     */
+    public void setNamespaces(final String namespacesString) {
+        final Properties properties = new Properties();
+        final StringReader sr = new StringReader(namespacesString);
+        try {
+            properties.load(sr);
+        }
+        catch (final IOException e) {
+            throw new MetafactureException("Failed to create namespace list");
+        }
+        finally {
+            if (sr != null) {
+                sr.close();
+            }
+        }
```

Comment:
`StringReader` is `AutoCloseable`, so this could be simplified with a `try`-with-resources statement:

```suggestion
        try (StringReader sr = new StringReader(namespacesString)) {
            properties.load(sr);
        }
        catch (final IOException e) {
            throw new MetafactureException("Failed to create namespace list");
        }
```